src/index.js

```diff
@@ -0,0 +1,36 @@
+const got = require('got');
+const HeaderGenerator = require('@petrpatek/headers-generator');
```

Comment:
Is this correct?

Reply:
Yeah, the official PR for header-generator should be done tonight. I wanted to prepare this as much we could until the PR is ready.